app/controllers/courses_controller.rb

```diff
@@ -237,6 +239,32 @@ def wiki_params
       .permit(:language, :project)
   end
 
+  def courses_wikis_params
+    params
+      .require(:course)
+      .permit(wikis: [:language, :project])
+  end
+
+  def update_courses_wikis
+    @course.ensure_home_wiki_in_courses_wikis
+    multi_wikis = courses_wikis_params[:wikis]
+    return if multi_wikis.nil?
+
+    # Used to differentiate deleted wikis
+    old_wikis = @course.wikis.to_a
```

Comment:
You don't need to convert an ActiveRecord relation to an array in most cases; you can do all the same things with the relation itself, including add or subject another relation or array to get an array back.